main.py

```diff
@@ -52,6 +63,8 @@ def get_config():
     }
     return config
 
+
 if __name__ == "__main__":
     import uvicorn
+
     uvicorn.run(app, host="0.0.0.0", port=8000)
```

Comment:
**Suggestion:** Avoid hardcoding values. Consider using environment variables or a configuration file for the host and port values in uvicorn.run.
```suggestion
if __name__ == "__main__":
    import uvicorn
    import os
    host = os.getenv('HOST', '0.0.0.0')
    port = int(os.getenv('PORT', 8000))
    uvicorn.run(app, host=host, port=port)
```